x-pack/plugins/security_solution/public/common/containers/source/use_data_view.tsx

```diff
@@ -102,8 +105,11 @@ export const useDataView = (): { indexFieldsSearch: (selectedDataViewId: string)
             },
           });
       };
-      searchSubscription$.current.unsubscribe();
-      abortCtrl.current.abort();
+      // only abort requests for ids already in progress
```

Comment:
is this right? i am exposing the `indexFieldsSearch` function and searching over a string array like:
```
activeDataViewIds.forEach((id) => indexFieldsSearch(id))
```

and it seems like the second request is canceling the first request

Reply:
Imo indexFieldsSearch should take a string or an array of ids, pass either on to the backend, and then make multiple calls in a loop to dataViewService.get() on the server side https://github.com/elastic/kibana/blob/main/x-pack/plugins/timelines/server/search_strategy/index_fields/index.ts#L103

Reply:
alternatively it would be pretty easy to refactor the get function to take an id string or an array of id strings i think https://github.com/elastic/kibana/blob/main/src/plugins/data_views/common/data_views/data_views.ts#L507

Reply:
but then you'd still have to change indexFieldsSearch anyway

Reply:
I am changing this logic in my PR coming, because we won't anymore get the data for all the data view but only for the one selected, can you both review it when my PR is ready? I found the same problem. And I do know that we can have more that on data view selected between `timeline` and `default` ;)